client/i18n/locales/english/intro.json

```diff
@@ -1695,6 +1695,72 @@
   "front-end-development": {
     "title": "Front End Development",
     "intro": ["placeholder"],
+    "chapters": {
```

Comment:
Do we want the welcome block from Quincy to be it's own chapter?
Right now it is listed as a module, but maybe it works for it to be its own chapter 🤔

Reply:
I think intro works for the title and the structure looks good to me 👍

Reply:
I added that [chapter here](https://github.com/freeCodeCamp/freeCodeCamp/pull/57151/files#diff-d5befb16f2e5667940f0cd15dba286e6a04c96343d8f6d2828e4f6332f0807a0R3) - I just called it `freecodecamp`, but I'm not sure that's the best.